com.github.SubhadeepJasu.pebbles.yaml

```diff
@@ -0,0 +1,53 @@
+app-id: com.github.SubhadeepJasu.pebbles
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+base: io.elementary.BaseApp
+base-version: juno-20.08
+command: com.github.subhadeepjasu.pebbles
+separate-locales: false
+finish-args:
+  - --share=network
+  - --socket=x11
```

Comment:
You already have fallback-x11, you probably meant `--socket=wayland` here

Reply:
Actually the `--socket=fallback-x11` is unnecessary as I think the app doesn't work on Wayland yet.

Reply:
Then you need at least `--share=ipc`

Reply:
It uses Gtk3, which uses Wayland, and I don't see where it forces X11 in the code. So what's the catch?

Reply:
Okay, I have added it to the manifest file.

Reply:
That mean having 
```
--socket=wayland
--socket=fallback-x11
--share=ipc
```